README.md

```diff
@@ -139,6 +141,12 @@ applied to the *virtualenv*.
 
 When the code is ready, submit a Pull Request.
 
+## Code Style
+
+* For Python code, adhere to PEP 8.
+* Prefer code that is readable over one that is "clever".
+* When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
```

Comment:
nit: Existing bullet points are `-`.  We may want to avoid the mix-up.

Reply:
Done